<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DlgEditGroupAndRole</class>
 <widget class="QDialog" name="DlgEditGroupAndRole">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>892</width>
    <height>533</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Edit Groups, Roles &amp; Folders</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTableView" name="tableView">
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>150</number>
     </attribute>
     <attribute name="horizontalHeaderMinimumSectionSize">
      <number>75</number>
     </attribute>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="grpSelected">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="title">
      <string>set in selected RDRs</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="2" column="1">
       <widget class="QLineEdit" name="edtFolder">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QCheckBox" name="chkGroup">
        <property name="text">
         <string>group:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="cmbGroup">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="2" rowspan="3">
       <widget class="QPushButton" name="btnSetSelected">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>&amp;Set</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QCheckBox" name="chkRole">
        <property name="text">
         <string>role:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="edtRole">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QCheckBox" name="chkFolder">
        <property name="text">
         <string>folder:</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="chkAlwaysEditable">
     <property name="text">
      <string>role always editable</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="chkNameEditable">
     <property name="text">
      <string>RDR name editable</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="btnHelp">
       <property name="text">
        <string>&amp;Help</string>
       </property>
       <property name="icon">
        <iconset resource="../lib/lib.qrc">
         <normaloff>:/lib/help/help.png</normaloff>:/lib/help/help.png</iconset>
       </property>
       <property name="default">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnCLose">
       <property name="text">
        <string>&amp;Close</string>
       </property>
       <property name="icon">
        <iconset resource="../lib/lib.qrc">
         <normaloff>:/lib/exit.png</normaloff>:/lib/exit.png</iconset>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../lib/lib.qrc"/>
  <include location="../lib/lib.qrc"/>
  <include location="../lib/lib.qrc"/>
  <include location="../lib/lib.qrc"/>
  <include location="../lib/lib.qrc"/>
  <include location="../lib/lib.qrc"/>
  <include location="../lib/lib.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>btnCLose</sender>
   <signal>clicked()</signal>
   <receiver>DlgEditGroupAndRole</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>881</x>
     <y>522</y>
    </hint>
    <hint type="destinationlabel">
     <x>287</x>
     <y>209</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkGroup</sender>
   <signal>toggled(bool)</signal>
   <receiver>cmbGroup</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>64</x>
     <y>403</y>
    </hint>
    <hint type="destinationlabel">
     <x>194</x>
     <y>404</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkRole</sender>
   <signal>toggled(bool)</signal>
   <receiver>edtRole</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>73</x>
     <y>430</y>
    </hint>
    <hint type="destinationlabel">
     <x>791</x>
     <y>432</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkFolder</sender>
   <signal>toggled(bool)</signal>
   <receiver>edtFolder</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>35</x>
     <y>453</y>
    </hint>
    <hint type="destinationlabel">
     <x>149</x>
     <y>450</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
